src/standard.ts

```diff
@@ -0,0 +1,661 @@
+    remove: unclassified,
+    edit: unclassified,
+  },
+  '/channels/*/publish/message/traits/correlationId': {
+    add: nonBreaking,
+    remove: breaking,
+    edit: breaking,
+  },
+  '/channels/*/publish/message/traits/correlationId/location': {
+    add: nonBreaking,
+    remove: breaking,
+    edit: breaking,
+  },
+  '/channels/*/publish/message/traits/correlationId/description': {
+    add: nonBreaking,
+    remove: nonBreaking,
+    edit: nonBreaking,
+  },
+  '/channels/*/publish/message/traits/schemaFormat': {
+    add: breaking,
+    remove: breaking,
+    edit: breaking,
+  },
+  '/channels/*/publish/message/traits/contentType': {
+    add: nonBreaking,
+    remove: breaking,
+    edit: breaking,
+  },
+  '/channels/*/publish/message/traits/name': {
+    add: nonBreaking,
+    remove: nonBreaking,
+    edit: nonBreaking,
+  },
+  '/channels/*/publish/message/traits/title': {
+    add: nonBreaking,
+    remove: nonBreaking,
+    edit: nonBreaking,
+  },
+  '/channels/*/publish/message/traits/summary': {
+    add: nonBreaking,
+    remove: nonBreaking,
+    edit: nonBreaking,
+  },
+  '/channels/*/publish/message/traits/tags': {
+    add: nonBreaking,
+    remove: nonBreaking,
+    edit: nonBreaking,
+  },
+  '/channels/*/publish/message/traits/externalDocs': {
+    add: nonBreaking,
+    remove: nonBreaking,
+    edit: nonBreaking,
+  },
+  '/channels/*/publish/message/traits/bindings': {
+    add: unclassified,
+    remove: unclassified,
+    edit: unclassified,
+  },
+  '/channels/*/publish/message/traits/examples': {
+    add: nonBreaking,
+    remove: nonBreaking,
+    edit: nonBreaking,
+  },
+  '/channels/*/publish/message/description': {
+    add: nonBreaking,
+    remove: nonBreaking,
+    edit: nonBreaking,
+  },
+  '/channels/*/publish/message/payload': {
+    add: unclassified,
+    remove: unclassified,
+    edit: unclassified,
+  },
+  '/channels/*/parameters': {
+    add: breaking,
+    remove: breaking,
+    edit: breaking,
+  },
+  '/channels/*/parameters/*': {
+    add: nonBreaking,
+    remove: breaking,
+    edit: breaking,
+  },
+  '/channels/*/parameters/*/description': {
+    add: nonBreaking,
+    remove: nonBreaking,
+    edit: nonBreaking,
+  },
+  '/channels/*/parameters/*/schema': {
+    add: unclassified,
+    remove: unclassified,
+    edit: unclassified,
+  },
+  '/channels/*/parameters/*/location': {
+    add: breaking,
+    remove: breaking,
+    edit: breaking,
+  },
+  '/components': {
+    add: nonBreaking,
+    remove: nonBreaking,
+    edit: nonBreaking,
+  },
+  '/tags': {
+    add: nonBreaking,
+    remove: nonBreaking,
+    edit: nonBreaking,
+  },
+  '/externalDocs': {
+    add: nonBreaking,
+    remove: nonBreaking,
+    edit: nonBreaking,
+  },
+};
+
+export type StandardType = typeof STANDARD;
+
+/**
+ * The standard object
+ */
+export const standard: StandardType = Object.assign(
+  Object.create(null),
+  STANDARD
+); // standard object without the prototype chain
```

Comment:
Why do you wanna create standard object without Object prototype? Could you explain?

Reply:
Having an object without a prototype chain will prevent us from accessing the objects which are inside the prototype chain.

Suppose we provide a string `__proto__` inside our classifier function, instead of giving `undefined`, we will get the proto object present in the prototype chain. Though it's highly unlikely that we will have these types of "attacks", but I have opted to go with object without prototype just as a safety measure.

I hope that it answers your question :). Let me know if you have any more queries.

Reply:
Ok I understand :) Thanks for explanation! However, I don't think that we should handle such a cases, because they are too rare or even absent - also we only use in the standard object the keys started by `/` char so we "always" will have unique keys. I only worry  about if someone will want to check if given property exist by `hasOwnProperty` function, then he/she will have to run the function as:

```js
Object.prototype.hasOwnProperty.call(standard, someKey).
```

I think that it can create unnecessary problems.

Reply:
Yeah, you are right. Should we move back to regular objects then? Like you said, all properties will have "/" in the start, so I don't think we will have any problem in the future with this.

Reply:
I think that we should :)

Reply:
On it.